starter-code/lab-javascript-funtions-arrays-EXERCISES.js

```diff
@@ -0,0 +1,117 @@
+//First Iteration
+
+function maxOfTwoNumbers(first, second){
+  if (first>second){
+  console.log(first);
+  }
+  else ("");{
+  console.log(second);
+}
+}
+maxOfTwoNumbers (2 , 6);
+
+//Second Itaration
+// No me sale y no sé por qué: Tengo definida la array words, el siguiente paso veo que sea distinto de cero, obtengo la array válidad distinta de cero, recorro la array y pido que me devuelva el sumatorio partido la longitud y digo que me imprima el resultado de la funcion... pues nada, me sale undefined :(
+
+var words = [
+	"seat",
+	"correspond",
+	"linen",
+	"motif",
+	"hole",
+	"smell",
+	"smart",
+	"chaos",
+	"fuel",
+	"palace"
+];
+
+function avg(words) {
+  if (words.length === 0 ) {
+    return;
+  }
+  for (var sum=0, i=0; i <= words.length; i++) {
+    sum += words[i];
+     }
+  return sum/words.length;
+
+}
+console.log(avg([]));
+
+// Third Itaration
+var words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+var differ = 0;
+//¿for(var i=0; esto es una variante de i = 0?
```

Comment:
sí, declaras una variable para el bucle for, aquí lo que harías es sobre escribir una variable que esté por encima en el código. Siempre declaralá dentro del bucle